### PR TITLE
Add instructions to install dev version

### PR DESCRIPTION
**Pull request by CGMossa**
Adds instructions to install the dev version everywhere in vscode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,27 +22,36 @@ Currently, it has an LSP component and a corresponding VS Code(ium) extension.
...
 1. Clone this repository locally
 2. Open it in VS Code; it's needed to run the extension
 3. In the `addons/vscode` subdirectory:
     1. Run `npm install` to install extension dependencies
     2. Run `npm run compile` to build the extension
 4. Run through the development cycle once to initialize and test everything
+5. If you want to install the dev version of the extension: After finishing the
+last step in Development Cycle, then press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>,
+and choose `Developer: Install Extension from Location...` and choose the
+the root directory for the extension. There will not be any messages, but
```
Comment (Petrosz007) [inline README.md:43]:
```suggestion
the directory for the extension: `addons/vscode/`. There will not be any messages, but
```

This wasn't clear to me, I kept selecting the root of the whole repo 😅

Comment (CGMossa) [reply README.md:43]: Oh you're right. That's what I actually did. Ups.

File: README.md [hunk 2]
```diff
@@ -20,27 +22,36 @@ Currently, it has an LSP component and a corresponding VS Code(ium) extension.
...
 
 ### First time setup
+
 1. Clone this repository locally
 2. Open it in VS Code; it's needed to run the extension
 3. In the `addons/vscode` subdirectory:
     1. Run `npm install` to install extension dependencies
     2. Run `npm run compile` to build the extension
 4. Run through the development cycle once to initialize and test everything
+5. If you want to install the dev version of the extension: After finishing the
```
Comment (nvarner) [inline README.md:40]:
```suggestion
5. (Optional: install the dev version of the extension) After finishing the
```

Break long sentence into two smaller units

File: README.md [hunk 3]
```diff
@@ -20,27 +22,36 @@ Currently, it has an LSP component and a corresponding VS Code(ium) extension.
...
+last step in Development Cycle, then press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>,
```
Comment (nvarner) [inline README.md:41]:
```suggestion
5. If you want to install the dev version of the extension:
Press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>,
```

Readers (hopefully) go step-by-step, so they will have already done this.

File: README.md [hunk 4]
```diff
@@ -20,27 +22,36 @@ Currently, it has an LSP component and a corresponding VS Code(ium) extension.
...
+and choose `Developer: Install Extension from Location...` and choose the
+the directory for the extension: `addons/vscode/`. There will not be any messages, but
```
Comment (nvarner) [inline README.md:43]:
```suggestion
and choose `Developer: Install Extension from Location...` and choose the
directory for the extension, `addons/vscode/`. There will not be any messages, but
```

Remove double "the" and replace colon with a comma

**Review by nvarner — CHANGES_REQUESTED** [review]
Good info here. Just a few small language comments.

**Review by nvarner — APPROVED** [review]

**Comment by nvarner** [comment]
:+1:

**Pull request merged**